query-engine/driver-adapters/src/proxy.rs

```diff
@@ -531,12 +531,12 @@ impl TryFrom<JSResultSet> for QuaintResultSet {
 
 impl CommonProxy {
     pub fn new(object: &JsObject) -> napi::Result<Self> {
-        let flavour: JsString = object.get_named_property("flavour")?;
+        let provider: JsString = object.get_named_property("provider")?;
```

Comment:
I am just wondering, how complicated it would be just for 1 release add fallback for `flavour` if `provider` is not found, just to give TiDB third party adapter enough time to adjust?

Reply:
(Update: I had to check for the property existence first, as otherwise Napi.rs would panic. I've left a comment in the code describing this).